studio/components/interfaces/Organization/TeamSettings/MembersView.tsx

```diff
@@ -126,13 +126,17 @@ const MembersView = () => {
                               <span className="flex rounded-full border-2 border-border-secondary-light p-2 dark:border-border-secondary-dark">
                                 <IconUser size={20} strokeWidth={2} />
                               </span>
-                            ) : (
+                            ) : !x?.username?.includes('@') ? (
```

Comment:
is this a correct check though or was there an update in the /members endpoint? the username doesn't include the "@" symbol
<img width="161" alt="image" src="https://user-images.githubusercontent.com/19742402/208372803-13999a62-cc1a-4628-8a09-e816b95f0062.png">

Reply:
anyone got better ideas?

I commented on the same thing in someone elses PR which was dealing with the same thing.
Can we check what auth provider the user is using?